src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -224,6 +224,7 @@ public void deletePerson(Person target) {
 
     @Override
     public void deleteEvent(Event target) {
+        reminders.deleteReminderOfEvent(target);
         calendar.removeEvent(target);
```

Comment:
Might want to consider swapping them around, calendar.removeEvent can throw a event not found exception. By swapping, this would implicitly confirm that event has been found and deleted before deleting reminder of said event.